keyboards/gboards/readme.md

```diff
@@ -0,0 +1,39 @@
+# gBoards Monorepo
+
+Ever wanted to do some crazy stuff with combos or onboard steno? Here's a big repo for you to include into your 
+keymaps. You may need to do some configuration on your part, for the documentation hit up [docs.gboards.ca](http://docs.gboards.ca/)
+
+## Building
+
+You can find all of my keyboards under the k/ folder. Make with 
+
+```make gboards/k/gergoplex:default:dfu``` or similar```
```

Comment:
```
`make gboards/k/gergoplex:default:dfu` or similar
```

(several codeblocks here use triple-backtick rather than single)

Reply:
Changed triple backticks to single backticks